.github/workflows/emscripten.yml

```diff
@@ -0,0 +1,104 @@
+name: Test Emscripten/Pyodide build
+
+on:
+  schedule:
+    # Nightly build at 3:42 A.M.
+    - cron: "42 3 */1 * *"
+  push:
+    branches:
+      - main
+      # Release branches
+      - "[0-9]+.[0-9]+.X"
+  pull_request:
+    branches:
+      - main
+      - "[0-9]+.[0-9]+.X"
+  # Manual run
+  workflow_dispatch:
+
+env:
+  FORCE_COLOR: 3
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  check_build_trigger:
+    name: Check build trigger
+    runs-on: ubuntu-latest
+    if: github.repository == 'scikit-learn/scikit-learn'
+    outputs:
+      build: ${{ steps.check_build_trigger.outputs.build }}
+    steps:
+      - name: Checkout scikit-learn
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
Sorry to go back on forth on this but I personally would rather not pin official GitHub actions i.e. `actions/...`, for example see in https://github.com/scikit-learn/scikit-learn/pull/29203#issuecomment-2152826986.

In other words, I would only pin third-party actions in package generating workflows.

Reply:
Ok to only pin third-party actions, we probably need to write down our policy somewhere then.

Sorry for the back and forth.